ISISComputingGroup/EPICS-IOC_Test_Framework
# Ticket 3991: Added options required for beckhoff tests

See https://github.com/ISISComputingGroup/IBEX/issues/3991

To test:
* Create IOC tests in a separate folder and point the IOC test framework at it
* Run a command line emulator and specify wait, confirm it does wait
* Both of these are done in https://github.com/ISISComputingGroup/BeckhoffPLCCode/pull/19
